### ISSUE TITLE
[BUG] Old Search bar for Rules Page

### ISSUE THREAD

**Issue by karanverma-qasource** [elastic/security-docs#1912]
**Description**
Old Search bar for Rules Page.

**Actual Result and Guide links**

- Rules Page : https://security-docs_1859.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#detection-engine-ui

![image](https://user-images.githubusercontent.com/91867110/165446049-69e0dd62-e194-46d7-beaf-3d15d0231532.png)

**Expected Result**

![image](https://user-images.githubusercontent.com/91867110/165446117-bbb6e440-e8fd-42a6-8365-adf295d1cda2.png)

**Comment by joepeeples** [comment]
This is a duplicate of https://github.com/elastic/security-docs/issues/1876, which was resolved. Once https://github.com/elastic/security-docs/pull/1843 is merged, the screenshot mentioned above will be updated with the new search bar.

**Comment by joepeeples** [comment]
Resolved by https://github.com/elastic/security-docs/pull/1843.
